packages/openssl/build.sh

```diff
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://www.openssl.org/
 TERMUX_PKG_DESCRIPTION="Library implementing the SSL and TLS protocols as well as general purpose cryptography functions"
 TERMUX_PKG_LICENSE="BSD"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.1.1k
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_VERSION=1.1.1h
```

Comment:
Why downgrading openssl?

Reply:
I'm sorry it's mistake, I'll revert this package.